utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -60,9 +60,13 @@ pub struct BenchmarkCmd {
 	#[clap(long, default_value = "1")]
 	pub external_repeat: u32,
 
-	/// Print the raw results.
-	#[clap(long = "raw")]
-	pub raw_data: bool,
+	/// Print the raw results in JSON format.
+	#[structopt(long = "json")]
+	pub json_output: bool,
+
+	/// Print the raw results in JSON format. Implies `--json`.
+	#[structopt(long = "json-file", parse(from_os_str))]
```

Comment:
We don't use structopt anymore.

And this should be set as `conflicts-with=json`? 
```suggestion
	#[clap(long, parse(from_os_str))]
```

Reply:
Okay I see, this was changes just some days ago :laughing: